components/ChallengeMaterial.tsx

```diff
@@ -139,7 +139,11 @@ export const ChallengeTitleCard: React.FC<ChallengeTitleCardProps> = ({
 export const ChallengeQuestionCard: React.FC<ChallengeQuestionCardProps> = ({
   currentChallenge
 }) => {
-  const diff = _.get(currentChallenge, 'submission.diff', false)
+  //temp fix while CLI tool is getting fixed
```

Comment:
could use a comment on what the regex is filtering out for. In the industry, regexes should always be accompanied by a comment.